runtime/development/src/lib.rs

```diff
@@ -1101,10 +1132,29 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
-	AllPallets,
-	(),
+	AllPalletsWithSystem,
+	SchedulerMigrationV3,
 >;
 
+// Migration for scheduler pallet to move from a plain Call to a CallOrHash.
+pub struct SchedulerMigrationV3;
```

Comment:
Not sure if it is needed for this stateless runtime, but should not harm.

Reply:
I think unless/until we have some sort of persistent development chain running somewhere, this isn't needed.

If it makes sense to add/remove the migrations here at the same time as elsewhere let's go ahead and do it, though, just to keep things consistent.